share/completions/pygmentize.fish

```diff
@@ -1,8 +1,12 @@
 function __fish_print_pygmentize
-    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*).$' '$1' | string trim)
+    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*)^\.' '$1' | string trim)
```

Comment:
I find it cleaner when there are no periods.

But, I couldn't let `string replace -r '^(.*).$' '$1'` stay because it messed with the completions. E.g. `cpp` becoming `cp`.